client/StringsManager.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+ * StringsManager.cpp, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+#include "StdInc.h"
+#include "StringsManager.h"
+
+#include "../lib/JsonNode.h"
+#include "../lib/filesystem/ResourceID.h"
+
+std::unordered_map<std::string, std::string> StringsManager::lookupDictionary;
+
+void StringsManager::init()
+{
+	JsonNode stringsJson{{"strings/en.json", EResType::TEXT}};
```

Comment:
Please take a look onto

	localizedTexts = JsonNode(ResourceID("config/translate.json", EResType::TEXT));

Reply:
nice! didn't find this file with quick glance